apps/sap-commerce-cloud-with-air/frontend/src/components/DialogClass.tsx

```diff
@@ -281,13 +288,15 @@ export default class Dialog extends React.Component<DialogProps, State> {
           )}
         </div>
         {isFieldTypeArray ? (
-          <Button
-            buttonType="primary"
-            icon="Done"
-            onClick={this.selectMultipleProductsClickEvent}
-            style={{ margin: '20px' }}>
-            Select Products
-          </Button>
+          <div>
```

Comment:
Are you just commenting out this button temporarily? Maybe add a TODO to enable it again?

Reply:
Yes i will do that now in a separate follow up branch